opencog/cogserver/scm/config.scm

```diff
@@ -44,6 +44,7 @@
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/sureal"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/lg-dict"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/planning"))
+(add-to-ltdl-path (string-append (getcwd) "/opencog/scm/opencog/learning"))
```

Comment:
yes, except that there is no ltdl at that location!  Do this:
```
ls -la build/opencog/scm/opencog
```
and you will see that there is no such file.  LTDL stands for "libtool dynamic library" and it is a convenience utility for working with dynamically loadable libraries.

All of the pattern miner shard objects are located in `build/opencog/learning/PatternMiner` ant the only one that seems to be an LTDL is the strangely named `liblibguile-patternminer.so` (perhaps one too many libs in there, eh?)